### ISSUE TITLE
IPFS updater does not install fs-repo-migrations

### ISSUE THREAD

**Issue by traumschule** [ipfs/ipfs-docs#824]
After upgrading ipfs-go from 0.6 to 0.9 the service fails to start:

```
Jun 30 06:22:23 host ipfs[2871]: Initializing daemon...
Jun 30 06:22:23 host ipfs[2871]: go-ipfs version: 0.9.0
Jun 30 06:22:23 host ipfs[2871]: Repo version: 11
Jun 30 06:22:23 host ipfs[2871]: System version: amd64/linux
Jun 30 06:22:23 host ipfs[2871]: Golang version: go1.16.5
Jun 30 06:22:23 host ipfs[2871]: Found outdated fs-repo, migrations need to be run.
Jun 30 06:22:23 host ipfs[2871]: Run migrations now? [y/N] Not running migrations of fs-repo now.
Jun 30 06:22:23 host ipfs[2871]: Please get fs-repo-migrations from https://dist.ipfs.io
Jun 30 06:22:23 host ipfs[2871]: Error: fs-repo requires migration
Jun 30 06:22:23 host systemd[1]: ipfs.service: Main process exited, code=exited, status=1/FAILURE
Jun 30 06:22:23 host systemd[1]: ipfs.service: Failed with result 'exit-code'.
Jun 30 06:22:26 host systemd[1]: ipfs.service: Service RestartSec=3s expired, scheduling restart.
Jun 30 06:22:26 host systemd[1]: ipfs.service: Scheduled restart job, restart counter is at 5.
Jun 30 06:22:26 host systemd[1]: Stopped ipfs.
Jun 30 06:22:26 host systemd[1]: ipfs.service: Start request repeated too quickly.
Jun 30 06:22:26 host systemd[1]: ipfs.service: Failed with result 'exit-code'.
Jun 30 06:22:26 host systemd[1]: Failed to start ipfs.
Jun 30 06:29:06 host systemd[1]: ipfs.service: Start request repeated too quickly.
Jun 30 06:29:06 host systemd[1]: ipfs.service: Failed with result 'exit-code'.
Jun 30 06:29:06 host systemd[1]: Failed to start ipfs.
```

Remedy: make sure that `fs-repo-migrations` is installed, or warn the administrator with instructions or offer to install it.

**Comment by johnnymatthews** [comment]
Thanks for raising this issue @traumschule. I've forwarded this to the IPFS team to figure out what's going on.

**Comment by johnnymatthews** [comment]
This error is likely due to MacOS not allowing IPFS to download and run unsigned executables. This has been fixed in [v0.10.0-rc1](https://github.com/ipfs/go-ipfs/releases/tag/v0.10.0-rc1)

**Comment by traumschule** [comment]
This issue 

> This error is likely due to MacOS not allowing IPFS to download and run unsigned executables.

Not in my case (Debian), however glad you fixed it.
